wrapper/src/main/java/software/amazon/jdbc/HostSpec.java

```diff
@@ -212,7 +212,7 @@ public String toString() {
 
   @Override
   public int hashCode() {
-    return Objects.hash(this.host, this.port, this.availability, this.role, this.weight, this.lastUpdateTime);
```

Comment:
availability, role, weight, and update time are all mutable - we should avoid using mutable fields in the hashcode implementation